crates/gradbench/src/main.rs

```diff
@@ -119,6 +120,12 @@ enum Commands {
         #[command(subcommand)]
         command: RepoCommands,
     },
+
+    /// Perform useful operations on the log files produced by `gradbench run`.
+    Log {
+        #[command(subcommand)]
+        command: LogCommands,
+    },
```

Comment:
A bit nitpicky, but can you move this above the `Repo` variant? I want to keep `Repo` at the bottom since it's the only category that really has different semantics from all the rest (it checks to see if there's a `.git` folder in the current working directory).